.github/workflows/build.yml

```
@@ -35,7 +38,7 @@ jobs:
           leanproject up
 
       - name: generate docs
-        run: mathlib/scripts/deploy_docs.sh "mathlib" ".." "mathlib"
+        run: deploy_docs.sh "mathlib" ".." "mathlib"
```

```suggestion
        run: ./deploy_docs.sh "mathlib" ".." "mathlib"
```

And CI has already found an error!